lib/Service/DirectoryService.php

```diff
@@ -105,7 +107,7 @@ private function getDirectoryFromListing(Listing|array $listing): array
 		if (isset($listing['publicationTypes']) && is_array($listing['publicationTypes'])) {
 			foreach ($listing['publicationTypes'] as &$publicationType) {
 				// Convert publicationType to array if it's an object
-				if ($publicationType instanceof \JsonSerializable) {
+				if ($publicationType instanceof JsonSerializable) {
```

Comment:
Why here check on instanceof JsonSerializable and further down.. (see other comment)

Reply:
I did not change anything here, i don't know, not relevant for this pr